web/yo/app/scripts/controllers/gene.js

```diff
@@ -618,6 +617,18 @@ angular.module('oncokbApp')
                         evidencesAllUsers[userName].deletedEvidenceModels.push(['mutation', mutation]);
                         continue;
                     }
+                    if (mutation.name_review.get('added')) {
+                        var tempEvidences = formSectionEvidencesByType('mutation', mutation);
```

Comment:
put this chunk inside if statement to a function to reuse for mutation/tumor/treatment